src/app/modules/group/pages/current-user/current-user.component.html

```diff
@@ -1,6 +1,6 @@
 <ng-container *ngIf="currentUser$ | async as currentUser">
   <div class="alg-page-header">
-    <div class="alg-page-header-title">{{ currentUser.firstName }} {{ currentUser.lastName }}</div>
+    <div class="alg-page-header-title --large">{{ currentUser.firstName }} {{ currentUser.lastName }}</div>
```

Comment:
I meant: shouldn't all titles be "large"?

Reply:
Fixed